beestation.dme

```diff
@@ -574,9 +574,9 @@
 #include "code\datums\looping_sounds\machinery_sounds.dm"
 #include "code\datums\looping_sounds\weather.dm"
 #include "code\datums\mapgen\_MapGenerator.dm"
+#include "code\datums\mapgen\CaveGenerator.dm"
 #include "code\datums\mapgen\JungleGenerator.dm"
 #include "code\datums\mapgen\biomes\_biome.dm"
-#include "code\datums\mapgen\CaveGenerator.dm"
```

Comment:
Hmm wait this doesn't really remove but move it still there is no reason to do that at all so just drop that change in the dme file and only keep the new added nanite protocol i guess this probably happened cause of some problems with vsc when adding the new relative path

Reply:
So just remove 577 and put in the path back at 579

Reply:
Thanks for that didn't notice my mistake, Think I Did everything you suggested besides adding his names to commits author (Don't know how to change the author of an already done commit. I did however add it to the changelog i believe)